docs/GRAPHITE_SETUP.md

```
@@ -0,0 +1,221 @@
+git --version
+```
+
+If needed, update via Homebrew:
+
+```bash
+brew install git
+```
+
+## Authentication
+
+1. Visit https://app.graphite.com/activate and sign in with your GitHub account
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)